HPC/README.md

```diff
@@ -438,6 +439,91 @@ Performance:        6.149        3.903
 Finished mdrun on rank 0 Fri May 12 22:18:17 2023
 ```
 
+## Developer guide
```

Comment:
So, removing the Open MPI installation w ACfL, the other instructions you show in this PR is the installation of ACfL. But the HPC Readme page already has Compilers section too. Can you just add the delta from here to those respective sections?

Reply:
This section https://github.com/aws/aws-graviton-getting-started/tree/main/HPC#compilers has some details on compiler installation/ setup. Can you just add the delta to it?

Also, I don't think we need to say "Developer guide". This page and Graviton Getting Started Guide is for developers :-).